lib/segment/src/data_types/vectors.rs

```diff
@@ -264,6 +264,10 @@ impl<T: PrimitiveVectorElement> TypedMultiDenseVector<T> {
     }
 
     pub fn len(&self) -> usize {
```

Comment:
Do we actually need this `len`? I propose to get flattened vectors len from flattened vectors